ktor-client/ktor-client-cio/jvm/test/io/ktor/client/engine/cio/CIOSpecificHttpsTest.kt

```diff
@@ -143,4 +146,33 @@ class CIOSpecificHttpsTest : TestWithKtor() {
             }
         }
     }
+
+    @Test
+    fun testGetServerTrusted() {
+        testWithEngine(CIO) {
+            config {
+                engine {
+                    https {
+                        trustManager = object : X509TrustManager {
+                            override fun checkClientTrusted(chain: Array<out X509Certificate>?, authType: String?) {
+                            }
+
+                            override fun checkServerTrusted(chain: Array<out X509Certificate>?, authType: String?) {
+                            }
+
+                            override fun getAcceptedIssuers(): Array<X509Certificate> {
+                                return emptyArray()
+                            }
+                        }
+                    }
+                }
+            }
+            test { client ->
+                assertEquals(
+                    HttpStatusCode.MethodNotAllowed,
+                    client.get("https://pt.api.sandbox.npay.eu/token/Grant").status
```

Comment:
Should we have an external URL in our tests?

Reply:
Hello, I work at the company responsible for this endpoint. I noticed you added this link in your tests and wondered about the same thing. For now, this URL is stable, but I can't promise it will be so in the future.

Reply:
Hey, @Stexxe, @jkauk-n, thanks for the report. We will drop the URL asap

Reply:
All right, thanks. This is a sandbox URL, so there's no harm in using it for now or blocking your release because of it.